src/ConfigTrait.php

```diff
@@ -59,7 +59,7 @@ public function readConfig($files = ['config.php'], string $format = 'php')
 
             $tempConfig = [];
 
-            switch (strtolower($format)) {
+            switch (mb_strtolower($format)) {
```

Comment:
I would not write this normally using mb_xxx. But ok to keep...

Reply:
format will never be multibyte so probably this also needs to be removed.